<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TorrentWidget</class>
 <widget class="QWidget" name="TorrentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>386</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">/* QGroupBox exposes the title as QString (even if internally its probably a QLabel) */
QGroupBox {
	font-weight: bold;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageGeneral">
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <property name="leftMargin">
        <number>48</number>
       </property>
       <property name="topMargin">
        <number>32</number>
       </property>
       <property name="rightMargin">
        <number>48</number>
       </property>
       <property name="bottomMargin">
        <number>32</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QLabel" name="labelTitle">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Torrent</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelMessage">
           <property name="text">
            <string>Select a torrent</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageTorrent">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tabFiles">
          <attribute name="title">
           <string>Files</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableView" name="fileTableView"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabInfo">
          <attribute name="title">
           <string>Info</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="downloadLabel">
               <property name="text">
                <string>Downloaded:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="TorrentProgressBar" name="torrentProgressBar">
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOn</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>425</width>
                <height>374</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QGroupBox" name="groupBox_1">
                 <property name="title">
                  <string>Transfer</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <layout class="QGridLayout" name="gridLayout_1" columnstretch="0,1,0,1,0,1">
                    <item row="0" column="0">
                     <widget class="QLabel" name="timeElapsedLabel">
                      <property name="text">
                       <string>Time Elapsed:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLabel" name="timeElapsedLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="2">
                     <widget class="QLabel" name="timeRemainingLabel">
                      <property name="text">
                       <string>Remaining:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="4">
                     <widget class="QLabel" name="wastedLabel">
                      <property name="text">
                       <string>Wasted:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="5">
                     <widget class="QLabel" name="peersLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="2">
                     <widget class="QLabel" name="uploadedLabel">
                      <property name="text">
                       <string>Uploaded:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="downloadedLabel">
                      <property name="text">
                       <string>Downloaded:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="5">
                     <widget class="QLabel" name="seedsLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="5">
                     <widget class="QLabel" name="wastedLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="4">
                     <widget class="QLabel" name="peersLabel">
                      <property name="text">
                       <string>Peers:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="3">
                     <widget class="QLabel" name="upLimitLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="2">
                     <widget class="QLabel" name="upSpeedLabel">
                      <property name="text">
                       <string>Upload Speed:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1">
                     <widget class="QLabel" name="downSpeedLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="4">
                     <widget class="QLabel" name="seedsLabel">
                      <property name="text">
                       <string>Seeds:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0">
                     <widget class="QLabel" name="downLimitLabel">
                      <property name="text">
                       <string>Down Limit:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="downSpeedLabel">
                      <property name="text">
                       <string>Download Speed:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="5">
                     <widget class="QLabel" name="shareRatioLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QLabel" name="downloadedLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="4">
                     <widget class="QLabel" name="shareRatioLabel">
                      <property name="text">
                       <string>Share Ratio:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="3">
                     <widget class="QLabel" name="upSpeedLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="1">
                     <widget class="QLabel" name="downLimitLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="3">
                     <widget class="QLabel" name="uploadedLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="3">
                     <widget class="QLabel" name="timeRemainingLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="2">
                     <widget class="QLabel" name="upLimitLabel">
                      <property name="text">
                       <string>Up Limit:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="0">
                     <widget class="QLabel" name="statusLabel">
                      <property name="text">
                       <string>Status:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="1">
                     <widget class="QLabel" name="statusLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_2">
                 <property name="title">
                  <string>General</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0,1">
                    <item row="0" column="0">
                     <widget class="QLabel" name="saveAsLabel">
                      <property name="text">
                       <string>Save As:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="3">
                     <widget class="QLabel" name="createdByLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="3">
                     <widget class="QLabel" name="completedOnLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0">
                     <widget class="QLabel" name="addedOnLabel">
                      <property name="text">
                       <string>Added On:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="1" colspan="3">
                     <widget class="QLabel" name="hashLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QLabel" name="totalSizeLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="3">
                     <widget class="QLabel" name="piecesLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="2">
                     <widget class="QLabel" name="piecesLabel">
                      <property name="text">
                       <string>Pieces:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1" colspan="3">
                     <widget class="QLabel" name="saveAsLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="5" column="1" colspan="3">
                     <widget class="QLabel" name="commentLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="5" column="0">
                     <widget class="QLabel" name="commentLabel">
                      <property name="text">
                       <string>Comments:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="1">
                     <widget class="QLabel" name="addedOnLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1">
                     <widget class="QLabel" name="createdOnLineEdit">
                      <property name="text">
                       <string notr="true">TextLabel</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="2">
                     <widget class="QLabel" name="completedOnLabel">
                      <property name="text">
                       <string>Completed On:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="2">
                     <widget class="QLabel" name="createdByLabel">
                      <property name="text">
                       <string>Created By:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="totalSizeLabel">
                      <property name="text">
                       <string>Total Size:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="createdOnLabel">
                      <property name="text">
                       <string>Created On:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="4" column="0">
                     <widget class="QLabel" name="hashLabel">
                      <property name="text">
                       <string>Hash:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="6" column="0">
                     <widget class="QLabel" name="magnetLinkLabel">
                      <property name="text">
                       <string>Magnet Link:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="6" column="1" colspan="3">
                     <widget class="QPlainTextEdit" name="magnetLinkEdit">
                      <property name="plainText">
                       <string notr="true">PlainTextEdit</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>0</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabPeers">
          <attribute name="title">
           <string>Peers</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableView" name="peerTableView"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabTrackers">
          <attribute name="title">
           <string>Trackers</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableView" name="trackerTableView"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabPieceMap">
          <attribute name="title">
           <string>Piece Map</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="TorrentPieceMap" name="torrentPieceMap" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageTorrentError">
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <property name="leftMargin">
        <number>48</number>
       </property>
       <property name="topMargin">
        <number>32</number>
       </property>
       <property name="rightMargin">
        <number>48</number>
       </property>
       <property name="bottomMargin">
        <number>32</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="labelErrorTitle">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string notr="true">Torrent Error</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelErrorMessage">
           <property name="text">
            <string notr="true">&lt;message&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TorrentPieceMap</class>
   <extends>QWidget</extends>
   <header>Widgets/TorrentPieceMap</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TorrentProgressBar</class>
   <extends>QProgressBar</extends>
   <header>Widgets/TorrentProgressBar</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
